cypress/e2e/ModelizerPage/SwitchView/delete.feature

```diff
@@ -0,0 +1,228 @@
+    Then I expect "[id^=\"image_\"]" exists
+    And  I expect "[id^=\"money_\"]" exists
+
+    When I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Text"
+    And  I expect "[data-cy=\"modelizer-text-view\"]" exists
+    And  I expect "[data-cy=\"file-label-default.plugin-test.js\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-label-leto-modelizer.config.json\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"inactive-tab\"]" is "default.plugin-test.js"
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"active-tab\"]" is "leto-modelizer.config.json"
+    And  I wait 1 second
+
+    When I click on "[data-cy=\"file-tabs-container\"] [data-cy=\"file-label-default.plugin-test.js\"]"
+    Then I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"active-tab\"]" is "default.plugin-test.js"
+    And  I expect active file content to contain "\"type\": \"image\""
+    And  I expect active file content to contain "\"type\": \"money\""
+
+    When I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Model"
+    And  I expect "[data-cy=\"modelizer-model-view-draw-root\"]" exists
+
+    When I click on "[id^=\"image_\"]"
+    And  I click on "[id=\"remove-component\"]"
+    Then I expect "[id^=\"money_\"]" exists
+    And  I expect "[id^=\"image_\"]" not exists
+
+    When I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Text"
+    And  I expect "[data-cy=\"modelizer-text-view\"]" exists
+    And  I expect active file content to contain "\"type\": \"money\""
+    But  I expect active file content to not contain "\"type\": \"image\""
+
+  Scenario: Remove object inside plugin file content (Text view) should remove related component (Model view)
+    #  NOTE: NOT WORKING if plugin file content is empty (error console -> TypeError: JSON.parse(...) is null - plugin-test.js)
+    Given I click on "[data-cy=\"component-definition-truck\"]"
+
+    When I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Text"
+    And  I expect "[data-cy=\"modelizer-text-view\"]" exists
+    And  I expect "[data-cy=\"file-label-default.plugin-test.js\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-label-leto-modelizer.config.json\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"inactive-tab\"]" is "default.plugin-test.js"
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"active-tab\"]" is "leto-modelizer.config.json"
+    And  I wait 1 second
```

Comment:
and -> when
put it in the next paragraph

Reply:
done